core/src/main/java/de/jplag/csv/comparisons/NameMapperIncrementalIds.java

```diff
@@ -0,0 +1,42 @@
+package de.jplag.csv.comparisons;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Maps the real names of submissions to random ids to anonymize the data.
+ */
+public class NameMapperIncrementalIds implements NameMapper {
```

Comment:
Are the duplicate name mappers still relics of the state before our discussion?
Right now, we just want simple incremental IDs in the order of the toplist.